src/test/java/space/obminyashka/items_exchange/end2end/LocationFlowTest.java

```diff
@@ -300,7 +300,7 @@ void createAreasInitFile_whenDataIsValid_shouldCreateFileAndReturnItsContent() t
                 .andReturn();
         final var responseContent = response.getResponse().getContentAsString();
         int parsedLocationsQuantity = (responseContent.length() - responseContent.replaceAll("(UUID_TO_BIN)", "").length()) / "UUID_TO_BIN".length();
-        assertEquals(2572, parsedLocationsQuantity, "Comparing unique locations number with result");
+        assertEquals(2894, parsedLocationsQuantity, "Comparing unique locations number with result");
```

Comment:
Can you describe that number with a variable that holds all newly created objects?

Areas: 24
Districts: ?
Cities: ?

and `var totalAmountOfNewlyCreatedEntities = totalAreas + totalDistricts + totalCities`